draft-ietf-quic-recovery.md

```diff
@@ -1496,6 +1496,11 @@ Invoked from DetectLostPackets when packets are deemed lost.
 
 Issue and pull request numbers are listed with a leading octothorp.
 
+## Since draft-ietf-quic-recovery-22
+
+- PTO should always send an ack-eliciting packet (#2895)
+- Unify the Handshake Timer with the PTO timer (#2648, #2658, #2886)
+
```

Comment:
(if #2916 gets merged)
- Move ACK generation text to transport draft (#2916)

Reply:
SG, added in anticipating of #2916 being merged.